integration/kubernetes/setup.sh

```diff
@@ -0,0 +1,54 @@
+#!/bin/bash
+#
+# Copyright (c) 2017 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -e
+
+SCRIPT_PATH=$(dirname "$(readlink -f "$0")")
+
+echo "Install Kubernetes"
+sudo bash -c "cat <<EOF > /etc/apt/sources.list.d/kubernetes.list
+deb http://apt.kubernetes.io/ kubernetes-xenial-unstable main
+EOF"
+curl -s https://packages.cloud.google.com/apt/doc/apt-key.gpg | sudo apt-key add -
+sudo -E apt update
+sudo -E apt install -y docker.io kubelet=1.6.7-00 kubeadm=1.6.7-00 kubectl=1.6.7-00
+sudo -E apt-mark hold kubelet kubeadm kubectl
+
+echo "Install Clear Containers, including dependencies"
+pushd "${SCRIPT_PATH}/../../.ci"
+./setup.sh
+popd
+
+echo "Install runc"
+go get -d github.com/opencontainers/runc
+pushd "${GOPATH}/src/github.com/opencontainers/runc"
+make
+sudo make install
```

Comment:
Can we install runc in a location that does not conflict with the default runc location. That way we do not break docker. Today the install breaks docker as runc is not backward compatible.